m4_sessions/1708-inning/group_two.md

```diff
@@ -21,16 +21,26 @@ What do we want people attending the session to learn or walk away with at the e
 * Being more actively aware of usability
 
 ### Session Outline:
-Intro to UX/UI (10 minutes)
-- need to come up with what this looks like
-Design Thinking Video (10 minutes or less)
-- need to find one or two great videos
+_We'll use a keynote slide deck to guide the session, which will be shared out at the end of the session_
+
+Intro to UX/UI (~10 minutes)
+* Facilitators introduce themselves, go around the room and do introductions and answer what ux/ui means to you
+
+Design Thinking Video (~15 minutes)
+* Video 1: https://www.youtube.com/watch?v=GFffb2H-gK0 (8:41 // A video produced by the Florida Hospital Innovation Labs covering the 5 different stages of design thinking, which are _empathy_, _reframe_, _brainstorming_, _prototype_, _testing_)
+* Video 2: https://www.youtube.com/watch?v=pXtN4y3O35M (5:51 // A quick five minute video analyzing IBM's design thinking approach to Agile development. IBM expands upon traditional design thinking by aiming at meeting the needs of large scale enterprises without sacrificing the personal focus of design thinking. The process starts with the end user and understanding what needs the user has that are not being met. Once a need is identified, the design team writes out as many design ideas as they can think of and slowly weed them out. The one that is left is then given to developers to create a working model.)
+
 Leader Group Exercise (10 minutes)
-- what we think is great/bad design with examples
-Small Group Exercise (10 minutes of research/talking, 5 minutes of sharing)
-- need to come up with "what" the groups will be doing
-Share principles of Simplicity, w/ slide show (10 minutes)
-Career Prep (15 minutes)
+* Session facilitators will each talk about what they think is great/bad design with examples
+
+Share principles of Simplicity (~10 minutes)
+* Visual examples of good design and bad design, with some explanations as to why
+
+Small Group Exercise (15 minutes of research/talking, 10 minutes of sharing)
+* Will break out into groups of 4 or 5 people, and have groups find examples of good design and bad design, and share with the rest of the class with explanations as to why
+
+Career Prep
+* Last slide of the slide deck, sharing next step for leveling up in your career
```

Comment:
I really like this connection to career prep and how they could be a huge value add to a team if they bring design thinking to the table.